src/typing_extensions.py

```diff
@@ -3524,16 +3525,41 @@ def _raise_attribute_error(self, name: str) -> Never:
         def __repr__(self) -> str:
             return self.__name__
 
-        def __getitem__(self, parameters):
-            if not isinstance(parameters, tuple):
-                parameters = (parameters,)
-            parameters = [
-                typing._type_check(
-                    item, f'Subscripting {self.__name__} requires a type.'
-                )
-                for item in parameters
-            ]
-            return typing._GenericAlias(self, tuple(parameters))
+        if sys.version_info >= (3, 11):
+            def __getitem__(self, parameters):
+                if not isinstance(parameters, tuple):
+                    parameters = (parameters,)
+                    parameters = [
```

Comment:
Why was this indented? This will work unexpectedly when passing multiple parameters.

Reply:
Thanks for the catch. This is a typo. It should be identical and to the variant before.

Reply:
This suggests we need more test cases, since this wasn't caught by the tests.

Reply:
My suggestion would then also be to raise a TypeError like the normal typing 3.12 variant.

Reply:
It seems hard to actually break this. But found a test that would have caught this.